protocol/messenger_linkpreview.go

```diff
@@ -137,6 +137,29 @@ func (m *Messenger) UnfurlURLs(httpClient *http.Client, urls []string) ([]common
 		httpClient = NewDefaultHTTPClient()
 	}
 
+	s, err := m.getSettings()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get settigs: %w", err)
+	}
+
+	// We use switch case, though there's most cases are empty for code clarity.
+	switch s.URLUnfurlingMode {
+	case settings.URLUnfurlingDisableAll:
+		return nil, fmt.Errorf("url unfurling is disabled")
+	case settings.URLUnfurlingEnableAll:
+		break
+	case settings.URLUnfurlingAlwaysAsk:
+		// This mode should be handled on the app side
+		// and is considered as equal to URLUnfurlingEnableAll in status-go.
+		break
+	case settings.URLUnfurlingSpecifyForSite:
```

Comment:
I understand the motivation @igor-sirotin, but well, I think we should only merge code that has some tangible value in production. This dummy code 🧟‍♂️ should never be reached because clients won't use the mode `URLUnfurlingSpecifyForSite`.

Reply:
👍 I agree, removed!